sfm_pc/management/commands/import_google_doc.py

```diff
@@ -37,8 +37,7 @@
 from organization.models import Organization, OrganizationAlias, \
     OrganizationClassification, OrganizationName, OrganizationRealStart
 
-from sfm_pc.utils import (import_class, get_osm_by_id, get_hierarchy_by_id,
-                          CONFIDENCE_MAP, execute_sql)
+from sfm_pc.utils import (import_class, CONFIDENCE_MAP, execute_sql)
```

Comment:
The changes in this file are primarily concerned with removing complicated location inference code and replacing it with simple retrieval of a related location by its "humane ID". The diff is pretty ugly, but hopefully this helps to frame the changes you see.

Reply:
I also flake8'd it.